### PR TITLE
[System Diagnostics and Health Check] HealthCheckPlaybookAnalysis script error fix

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Some variables are defined under an if statement and checked in the global scope. If the statement fails the whole script fails because they are not defined.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ApiModules/Scripts/CoreIRApiModule/1_41d0cf8a-0b47-4369-9359-ad66e9559f5a [hunk 1]
```diff
@@ -0,0 +1 @@
+"test_file"
```
Comment (moishce) [inline Packs/ApiModules/Scripts/CoreIRApiModule/1_41d0cf8a-0b47-4369-9359-ad66e9559f5a:1]: This file should be deleted

File: Packs/HealthCheck/ReleaseNotes/2_0_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### HealthCheckPlaybookAnalysis
+- Fixed 'copyDetected' not defined error
```
Comment (moishce) [inline Packs/HealthCheck/ReleaseNotes/2_0_3.md:4]:
```suggestion
- Fixed 'copyDetected' not defined error.
```

**Review by moishce — COMMENTED** [review]
It looks great
See my comments below.
and the docker image tag in the yml file is not updated.

**Comment by ennozdd** [comment]
Setup Environment fails but I believe it has nothing to do with the changes

**Pull request merged**